src/common/matrixproperties.jl

```diff
@@ -42,6 +42,7 @@ See also [`isisometry`](@ref) and [`is_right_isometry`](@ref).
 """ is_left_isometry
 
 function is_left_isometry(A::AbstractMatrix; isapprox_kwargs...)
+    iszero(min(size(A)...)) && return true
```

Comment:
I would say: `iszero(size(A, 2)) && return true` for `is_left_isometry` and `iszero(size(A,1))` for `is_right_isometry below`.